swap/swap.go

```diff
@@ -75,14 +75,10 @@ type Owner struct {
 
 // Params encapsulates param
 type Params struct {
```

Comment:
Why should we use this structure? What is the common denominator of all variables under `Params`?

Reply:
I thought of it as the set of parameters which the user can change to parametrize swap behavior.

Mostly stuff the node operator is interested in which are no details about anything Swarm but financially related (thresholds, amounts, cashing strategy, etc.)

Reply:
Ok. Params = parameters which can be set in the command line. Conversation may be resolved.